compiler/rustc_lint_defs/src/lib.rs

```diff
@@ -330,6 +330,10 @@ pub struct Lint {
     pub feature_gate: Option<Symbol>,
 
     pub crate_level_only: bool,
+
+    /// `true` if this lint should not be filtered out under any circustamces
+    /// (e.g. the unknown_attributes lint)
+    pub loadbearing: bool,
```

Comment:
Can we give this a more descriptive name, like `always_run` or `run_unconditionally` or `never_filter`?

Reply:
eval_always :smile: ?

Reply:
At least that would be consistent with the query equivalent 😉